packages/nextjs-template/components/DendronSearch.tsx

```diff
@@ -117,7 +126,7 @@ function DendronSearchComponent(
     }
     const out =
       qs === ""
-        ? NoteLookupUtils.fetchRootResults(notes)
+        ? NoteLookupUtils.fetchRootResults(notes as NotePropsDict)
```

Comment:
don't cast notes to something its not, if `fetchRootResults` works with `sections` type, then add it to the type of `fetchRootResults`